beetsplug/replaygain.py

```diff
@@ -251,7 +251,14 @@ def end_element_handler(name):
                 state['gain'] = state['peak'] = None
         parser.StartElementHandler = start_element_handler
         parser.EndElementHandler = end_element_handler
-        parser.Parse(text, True)
+
+        try:
+            parser.Parse(text, True)
+        except xml.parsers.expat.ExpatError:
+            raise ReplayGainError(u'bs1770gain returned malformed XML - this '
+                                  'is a bug in versions prior to v0.4.10, '
+                                  'please ensure that your version is up to '
+                                  'date')
```

Comment:
This would require a little less wrapping with the parentheses style where you wrap immediately after the (. For example:

```py
raise ReplayGainError(
    u'The bs1770gain tool produced malformed XML. '
    'Using version >=0.4.10 may solve this problem.'
)
```

Reply:
I've made the change to both the wrapping and the message itself, thanks! :+1: